src/Application/Migrations/__2018-06-21_fix-trigger-for-progress-recalculation.sql

```diff
@@ -0,0 +1,30 @@
+BEGIN;
+
+CREATE OR REPLACE FUNCTION update_ticket_progress()
+  RETURNS trigger AS
+$BODY$
+  BEGIN
+    IF TG_OP = 'DELETE' THEN
+      IF (OLD.parent_id IS NOT NULL) THEN
+        UPDATE tbl_ticket SET progress = ticket_progress(OLD.parent_id) WHERE id = OLD.parent_id;
+      END IF;
+	  RETURN NULL;
+    ELSE
+      UPDATE tbl_ticket SET progress = ticket_progress(NEW.iqd) WHERE id = NEW.id;
```

Comment:
"NEW.iqd" looks like a typo.

Reply:
Interesting.. why did my tests succeed then. Will test again.

Reply:
It's only in the migration, isn't it?

Reply:
Thanks, that explains it. So it must be happened on copying. I'm reworking the commit anyway.